tools/topology/sof-tgl-nocodec.m4

```diff
@@ -148,6 +213,7 @@ PCM_DUPLEX_ADD(Port0, 0, PIPELINE_PCM_1, PIPELINE_PCM_2)
 PCM_DUPLEX_ADD(Port1, 1, PIPELINE_PCM_3, PIPELINE_PCM_4)
 PCM_DUPLEX_ADD(Port2, 2, PIPELINE_PCM_5, PIPELINE_PCM_6)
 PCM_CAPTURE_ADD(DMIC, 6, PIPELINE_PCM_13)
+PCM_CAPTURE_ADD(EchoRef, 3, PIPELINE_PCM_7)
```

Comment:
FwEchoRef

we need to start differentiating between firmware generated echo reference and hardware-generated echo reference. If we don't we'll always have confusion on the topology.

FwEchoRef tells you the data is provided by firmware
HwEchoRef tells you the data is provided by external hardware.